.vib/appsmith/goss/vars.yaml

```diff
@@ -0,0 +1,27 @@
+binaries:
+  - java
+  - mongosh
+  - nginx
+  - node
+  - python
+  - render-template
+  - wait-for-port
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/appsmith
+      - /opt/bitnami/appsmith/conf
+      - /opt/bitnami/appsmith/logs
+      - /opt/bitnami/appsmith/tmp
+  - paths:
+      - /opt/bitnami/appsmith/backend/plugins
+      - /opt/bitnami/appsmith/editor
+      - /opt/bitnami/appsmith/templates/client
+files:
+  - paths:
+      - /opt/bitnami/appsmith/conf/docker.env
```

Comment:
Shouldn't we add `mode` check for `/opt/bitnami/appsmith/conf/docker.env` considering they are explicitly set in the Appsmith postunpack logic?